common/client-libs/validator-client/src/nym_api/mod.rs

```diff
@@ -96,6 +97,14 @@ pub trait NymApiClientExt: ApiClient {
         .await
     }
 
+    async fn get_mixnodes_described(&self) -> Result<Vec<DescribedMixNode>, NymAPIError> {
+        self.get_json(
+            &[routes::API_VERSION, routes::GATEWAYS, routes::DESCRIBED],
```

Comment:
didn't you mean to call mixnodes route as opposed to gateways?

Reply:
good catch 😅